__tests__/app.test.js

```diff
@@ -70,6 +70,20 @@ describe("app", () => {
 					});
 				});
 		});
+		test("Status 200 - articles have a comment_count property showing number of comments on article", () => {
+			return request(app)
+				.get("/api/articles")
+				.expect(200)
+				.then(({ body: { articles } }) => {
+					articles.forEach((article) => {
```

Comment:
If you're going to assert within a forEach always assert against the length as well, if it returns an empty array it'll be a false positive!

Reply:
Great tip - hadn't thought of that - thanks!